how-to-guides/neptune-docker/scripts/api_token.txt

```diff
@@ -0,0 +1 @@
+ANONYMOUS
```

Comment:
hmm, If it a secret why you put is as open text? :thinking:

Reply:
Because that's how you create an internal secret
https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets